docs/how-to-deploy-cp4waiops-daily-build.md

```diff
@@ -7,7 +7,12 @@ The procedure for deploying CP4WAIOps with daily build is very similar compare t
    - "Username": [Your Email address]
    - "Password": paste the api token of the account above
    - "Email": any email, valid or not, will work. This fields is mostly a hint to other people who may see the entry in the configuration  
-  
+   
+You can test the build repository credential using docker command below.
+```console
+docker login [build repository] -u [Your Email address] -p [API Token]
+docker pull [catalog image]
```

Comment:
May need to tell beginner users where to find the `[catalog name]`, otherwise, they may not be able to figure out what the value they have to fill in here. e.g.: to tell user run `oc get catalogsource blablabla` and look for the image by checking certain field.

Reply:
how to get the `[catalog image]` is already in the box note in the same section

Reply:
Can we move the above newly added paragraph to the end of this document, since this is the first time the [catalog image] is appeared. And the explanation of [catalog image] is after that now, which makes it a bit hard to follow for readers.